packages/react-native-editor/src/index.js

```diff
@@ -21,9 +21,18 @@ import {
 import { unregisterBlockType, getBlockType } from '@wordpress/blocks';
 
 const reactNativeSetup = () => {
-	// Disable warnings as they disrupt the user experience in dev mode
-	// eslint-disable-next-line no-console
-	console.disableYellowBox = true;
+	LogBox.ignoreLogs( [
+		'Require cycle:', // TODO: Refactor to remove require cycles
+		'lineHeight', // TODO: Remove lineHeight warning from Aztec
+		/**
+		 * TODO: Migrate to @gorhom/bottom-sheet or replace usage of
+		 * LayoutAnimation to Animated. KeyboardAvoidingView's usage of
+		 * LayoutAnimation collides with both BottomSheet and NavigationContainer
+		 * usage of LayoutAnimation simultaneously https://git.io/J1lZv,
+		 * https://git.io/J1lZY
+		 */
+		'Overriding previous layout animation',
+	] );
```

Comment:
`disableYellowBox` is deprecated. Additionally, its complete disablement likely leads to overlooking new warning and errors during feature development. Hopefully reenabling it and selectively filtering warnings is a better approach.

Reply:
Agreed. Total disabling is not prudent in the long term. 👍